index.html

```diff
@@ -2,13 +2,238 @@
+                  >Pick up groceries</span
+                >
+                <span class="remove h-3 w-3 cursor-pointer">
+                  <img
+                    class="absolute right-3 h-4 w-4"
+                    alt="cross-icon"
+                    title="cross-icon-image"
+                    src="images/icon-cross.svg"
+                  />
+                </span>
+              </label>
+            </li>
+            <li
+              class="list-item size-full h-16 list-none justify-between border border-blue-200 bg-blue-0 p-4 text-base text-blue-200"
+            >
+              <label class="list relative flex cursor-pointer items-center pt-1.5">
+                <input
+                  class="checkbox checked:bg-gradient-to-r peer mr-2.5 h-5 w-5 cursor-pointer appearance-none rounded-full border border-blue-200 from-blue-700 to-blue-800 checked:border-none checked:bg-center checked:bg-no-repeat"
+                  type="checkbox"
+                /><img
+                  src="./images/icon-check.svg"
+                  alt="Icon"
+                  class="icon-check relative right-7"
+                />
+                <span
+                  class="font-bold text-blue-400 peer-checked:text-blue-500 peer-checked:line-through"
+                  >Complete Todo App</span
+                >
+                <span class="remove h-3 w-3 cursor-pointer">
+                  <img
+                    class="absolute right-3 h-4 w-4"
+                    alt="cross-icon"
+                    title="cross-icon-image"
+                    src="images/icon-cross.svg"
+                  />
+                </span>
+              </label>
+            </li>
+          </ul>
+        </div>
+
+        <div
+          class="flex size-full h-14 justify-between rounded-b-lg border border-blue-200 bg-blue-0 p-4 text-base text-blue-500"
+        >
+          <div class="pt-1"><span>5</span> items left</div>
+          <div class="cursor-pointer">Clear Completed</div>
+        </div>
+
+        <div
+          class="mt-5 flex size-full h-12 justify-between rounded-lg bg-blue-0 p-4 text-base text-blue-500"
+        >
+          <div class="m-auto flex">
+            <label class="cursor-pointer hover:text-blue-400">
+              <input class="peer opacity-0" type="radio" name="filter" checked="" />
+              <span class="peer-checked:text-blue-600">All</span>
+            </label>
+            <label class="cursor-pointer hover:text-blue-400">
+              <input class="peer opacity-0" type="radio" name="filter" />
+              <span class="peer-checked:text-blue-600">Active</span>
+            </label>
+            <label class="cursor-pointer hover:text-blue-400">
+              <input class="peer opacity-0" type="radio" name="filter" />
+              <span class="peer-checked:text-blue-600">Completed</span>
+            </label>
+          </div>
```

Comment:
lets use nav here instead,also i think the use of peer here is unnecessary please correct that also clicking your links makes it more of focused than hover or active